### PR TITLE
Add username and password

### PR DESCRIPTION
**Pull request by cdupuis**

### CODE REVIEW

File: docker/ops.go [hunk 1]
```diff
@@ -173,8 +179,8 @@ func ProcessMessage(message *babashka.Message) (any, error) {
 			if err := json.Unmarshal([]byte(message.Args), &args); err != nil {
 				return nil, err
 			}
-
-			err := generate_sbom(message, args[0])
+			// TODO add username and password to inbound message
+			err := generate_sbom(message, args[0], "", "")
```
Comment (kipz) [inline docker/ops.go:183]: @cdupuis so we need to get the creds in to here somehow?

Comment (cdupuis) [reply docker/ops.go:183]: Yeah, but that is an area I don't understand as it deals with the incoming babushka message.

Comment (jonnystoten) [reply docker/ops.go:183]: I can have a look at this next

**Review by jonnystoten — APPROVED** [review]
LGTM

**Pull request merged**